internal/interface/crawler/crawler.go

```diff
@@ -34,10 +47,39 @@ func NewCrawler(db *sql.DB, articlesRepo repository.ArticleRepo) *Crawler {
 }
 
 type Configuration struct {
-	RootURL             string
+	RootURL			 string
 	DesiredArticleCount int
 }
 
+func (c *Crawler) GetUnvisitedURL() (string, error) {
+	rows, err := c.db.Query("SELECT URL FROM CrawlStatus WHERE Visited = false LIMIT 1;")
```

Comment:
May be it would be better to query multiple rows? So, if there are more than one url it is not reasonable to get urls one by one.